components/textbook-demo/PrerequisiteMaterialSection.vue

```diff
@@ -4,32 +4,21 @@
       Prerequisite material
     </h2>
     <div class="prerequisite-material-section__cards">
-      <article
+      <div
         v-for="{ title, description, progress, segment, url } in prerequisites"
         :key="title"
         class="prerequisite-material-section__card"
```

Comment:
this is the container for the cards, so, to improve readability and responsibility separation, this container should have the styles:
```
    align-items: strech;
    display: flex;
    padding: $spacing-05 $spacing-05 0;
```

and the card itself should have:
```
    background: $cool-gray-10;
    color: $cool-gray-80;
```

Reply:
Good catch! That wrapping level is unnecessary. I can move the style and `v-for` to the child component and get the same results indeed. 🙂